.dockerignore

```
@@ -0,0 +1,8 @@
+.github
+.pytest_cache
+.venv
+.vscode
+.env
+.env.test
+.gitignore
+env.template
```

```suggestion
.env.template
```

These are good things to include in the .dockerignore, though it's unimportant since your COPY commands avoid these files anyway.

Other things you could include:
`__pycache__` (which is present inside the todo_app folder, so this is the only one that could actually affect you), `.git`, `Dockerfile`, `README.md`